pyscriptjs/examples/antigravity.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <py-env>
 - paths:
-  - /antigravity.py
+  - antigravity.py
```

Comment:
Iirc, the way browsers will interpret this is to translate it to `./antigravity`.. In that case, I'd suggest changing to that.